DigitalLearningSolutions.Data/Services/CustomPromptsService.cs

```diff
@@ -4,6 +4,7 @@
     using System.Linq;

```

Comment:
This is getting quite long - I wonder if we could refactor this into two separate services, one for registration prompts and the other for admin fields.. Olivia is working on this too atm, so she could potentially do the refactor later.

Reply:
I agree with this, it's at a similar length to the UserDataService I've just split. @livzorn FYI, this split might come your way at the end of the admin fields work.

Reply:
@AlexJacksonDS ah yeah thanks, I'll keep that in mind!